playbooks/build-heat-stack.yml

```diff
@@ -26,6 +26,7 @@
 
     - name: Add host to the dynamic inventory
       add_host:
-        name: "{{ heat_stack.stack.outputs | selectattr('output_key', 'equalto', 'floating_ip') | map(attribute='output_value') | first }}"
+        name: "webserver"
```

Comment:
Instead of using hard coded name, you could use dynamic name:
```
add_host:
  name: "{{ item.name }}"
  ...
with_items:
  - "{{ heat_stack.stacks }}"
```
Here, the result will be `nginx-vm-stack`